ietf/doc/utils_search.py

```diff
@@ -150,7 +150,6 @@ def fill_in_document_table_attributes(docs, have_telechat_date=False):
         elif rel.relationship_id == "updates":
             l = d.updated_by_list
         l.append(rel_rfc_aliases[rel.source_id])
-        l.sort()
```

Comment:
This sort was in an odd place anyhow (repeatedly sorting a list as things were appended to it), but _not_ sorting is likely not safe - you are relying on the natural order coming back from `RelatedDocument.objects.filter` and that ordering is undefined (casual tests may show it returning things in the order they were inserted into the RelatedDocument table, but that is not guaranteed to remain true).

Reply:
an order_by on the query might be sufficient.

Reply:
So `interesting_doc_relations` also doesn't seem to sort, which is what we use elsewhere. Should it sort? What do I pass into `order_by`, I wasn't able to figure it out?

Reply:
I've made some complicated suggestions just typing into github's UI - there may be issues with them.